src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsremandandsentencingapi/legacy/model/LegacySentenceTypeGroupingSummary.kt

```diff
@@ -0,0 +1,8 @@
+package uk.gov.justice.digital.hmpps.hmppsremandandsentencingapi.legacy.model
+
+data class LegacySentenceTypeGroupingSummary(
+  val nomisSentenceTypeReference: String,
+  val nomisDescription: String,
+  val isIndeterminate: Boolean,
+  val recall: RecallType,
+)
```

Comment:
minor - but instead of having recall as mandatory and a RecallType of NONE.. would it be better to have recall as an optional?

+ maybe the var should be called recallType?

Reply:
I would have to use null rather than optional (optional isn't preferred by Kotlin)